server/utils/world/initializeWorldDataObject.ts

```diff
@@ -0,0 +1,51 @@
+import { Credentials } from "../../types/Credentials.js";
+import { errorHandler } from "../errorHandler.js";
+import { getDefaultKeyAssetImage } from "../getDefaultKeyAssetImage.js";
+import { DroppedAsset } from "../topiaInit.js";
+
+export const initializeWorldDataObject = async ({ credentials, world }: { credentials: Credentials, world: any }) => {
+  try {
+    let { assetId, urlSlug } = credentials
+    const sceneDropId = credentials.sceneDropId || assetId
+
+    const droppedAsset = await DroppedAsset.get(assetId, urlSlug, { credentials });
```

Comment:
This has a huge impact in performance, because it will try to initialize the app for every request, hence it is fetching the droppedAsset outside of the if statement. It should fetch inside of the if statement, otherwise every request will fetch the droppedAsset without need

Reply:
Good catch! I meant to move this into an if statement but forgot. Thanks!